docs/howto/krustlet-on-microk8s.md

```diff
@@ -9,6 +9,20 @@ MicroK8s and the Krustlet, on a single machine. `kubectl` is required but is ins
 as `microk8s.kubectl`. The following instructions use `microk8s.kubectl` for simplicity.
 You may use a standlone `kubectl` if you prefer.
 
+In order for the bootstrap authentication token to work, your kube-apiserver needs to have
+the `--enable-bootstrap-token-auth`
```

Comment:
please add "feature flag" after "the `--enable-bootstrap-token-auth`". `--enable-bootstrap-token-auth` is not a proper noun

Reply:
got it